systers_portal/templates/meetup/snippets/user-cell.html

```diff
@@ -0,0 +1,15 @@
+{% load staticfiles %}
+
+{% if user.profile_picture.name and user.profile_picture.name != "False" %}
+  <a href="{{ user.get_absolute_url }}">
+    <img src="{{ user.profile_picture_thumbnail.url }}" class="profile-pic img-square img-responsive crop_img" alt="{{ user }} profile picture"/>
```

Comment:
I too thought to remove img_crop as we don't need resizing any more, but when I removed this class and opened browser to test, in chrome it was fine, but in firefox resized images are looking a bit small http://127.0.0.1:8000/meetup/indore/members/
But with crop_img class all the images are looking same sized in both the browsers.

I tried resizing image in users/models.py but resizing effects only if I change any one dimension, for eg: 100x1000 will change the height but 100x100 will not change anything.